.circleci/config.yml

```diff
@@ -1,6 +1,23 @@
 version: 2.1
+commands:
+  push-image:
+    description: "push docker-image to registry"
+    parameters:
+      image-name:
+        type: string
+      image-url:
+        type: string
+    steps:
+      - run:
+          name: push << parameters.image-url >>
+          command: |
+            docker login -u $DOCKER_HUB_USER -p $DOCKER_HUB_PASSWORD
+            docker tag << parameters.image-url >> << parameters.image-url >>:$(cat ./<< parameters.image-name >>/version)
+            docker push << parameters.image-url >>:$(cat version)
+            docker push << parameters.image-url >>
```

Comment:
You push twice? First time with an explicit tag and the second time with `latest`?

Reply:
yes